schemas/TestCase.update.yaml

```diff
@@ -48,6 +48,10 @@ properties:
           type: string
         position:
           type: integer
+        attachments:
+          type: array
+          items:
+            type: string
```

Comment:
please add a description here and the previous pr

items - what is it? what inside - hash, URLs, etc?

Reply:
✅